docs/source/install/ewc.rst

```diff
@@ -41,7 +41,7 @@ replacing ``${EWC_LICENSE_KEY}`` with the license key you received when register
   ./install.sh --user=st2admin --password='Ch@ngeMe' --license=${EWC_LICENSE_KEY}
 
 To understand the full details of the installation procedure, or to install |ewc| manually, follow
-the installation guide for your Linux version: :doc:`/install/u16`, :doc:`/install/u18`, :doc:`/install/rhel7`,
+the installation guide for your Linux version: :doc:`/install/u16`, :doc:`/install/u18`,:doc:`/install/rhel8`, :doc:`/install/rhel7`,
```

Comment:
Nit picking here... Ubuntu goes 16, 18, but you make RHEL 8, 7. I don't care either way, just be consistent

Reply:
done